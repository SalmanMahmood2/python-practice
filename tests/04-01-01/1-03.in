Gala
Fuji

gala
Gala
FUJI
Fuji
