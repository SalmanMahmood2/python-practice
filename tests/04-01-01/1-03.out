Not in stock!
In stock!
Not in stock!
In stock!